landingzones/lz-healthcare/networking.bicep

```diff
@@ -171,6 +231,14 @@ module nsgWebApp '../../azresources/network/nsg/nsg-empty.bicep' = {
 }
 
 // Route Tables
+resource udr 'Microsoft.Network/routeTables@2021-02-01' = {
```

Comment:
Should the udr resource follow the pattern of nsg to have array of udrs?

Reply:
No it shouldn't since the routing will the same for all subnets.

Reply:
sounds good!